### PR TITLE
fixes for kubernetes 1.9

### PR DESCRIPTION
**Pull request by pstadler**

### CODE REVIEW

File: service/kubernetes/scripts/master.sh [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #!/bin/sh
 set -e
 
-kubeadm init --config /tmp/master-configuration.yml
+kubeadm init --config /tmp/master-configuration.yml \
+  --ignore-preflight-errors=Swap
```
Comment (Qambar) [inline service/kubernetes/scripts/master.sh]: What is the actual reason kubernetes looks for swap is on in preflight-checks ?

**Comment by pstadler** [comment]
These changes seem to work, but weave-net fails currently.

**Comment by pstadler** [comment]
Finally got a 1.9.1 cluster running 👍

File: service/kubernetes/scripts/master.sh (hunk 1 above)
Comment (pstadler) [reply service/kubernetes/scripts/master.sh]: They say using swap is unsupported, but it has been working nicely in the past and I still believe we should enable it for the time being.

**Pull request merged**